packages/admin-panel/src/VizBuilderApp/components/PreviewOptions.jsx

```diff
@@ -174,13 +181,13 @@ export const PreviewOptions = () => {
   };
 
   const handleChangeStartDate = date => {
-    const newDate = date ? date.toISOString() : null;
+    const newDate = date ? getIsoDateString(date) : null;
```

Comment:
If you wanted to get rid of a bit of repetition between start/end date handling, you could potentially make a function that accepts `date` as a date OR `null` OR `undefined` and returns either `null` or the date string

Reply:
Done! Tidier 🎉